src/connection/ConnectionDB.js

```diff
@@ -1,14 +1,17 @@
 const mongoose = require('mongoose');
 
+const{NODE_ENV, URI_PRODUCTION, URI_TEST} = process.env
 // Connection the dataBase
-const dataBaseConnect = () => {
-    const URI = process.env.URI;
-    mongoose.connect(
+const dataBaseConnect = async () => {
+    const URI = NODE_ENV === 'test'
+    ? URI_PRODUCTION
+    : URI_TEST;
+    await mongoose.connect(
         URI, {
         useNewUrlParser: true,
         useUnifiedTopology: true
-    }).then(db => console.log('**** CORRECT CONNECTION ****'))
-    .catch(err => console.log('***** CONNECTION ERROR ****', err));
+    }).then(database => console.log('**** CORRECT CONNECTION ****'))
+    .catch(error => console.log('***** CONNECTION ERROR ****', error.message));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    .catch((error) => console.log('***** CONNECTION ERROR ****', error.message));
```